endpoint/tests/test_endpoint_on_development.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -uo pipefail
+
+source endpoint/tests/common.sh
+
+# Download Legato and build the tools
+repo init -u git://github.com/legatoproject/manifest -m legato/releases/${LEGATO_VERSION}.xml
```

Comment:
Don't check out Legato AF from GitHub in test script(s). Integrate into build system instead.